### ISSUE TITLE
rsync__args - undefined method `strip' 

### ISSUE THREAD

**Issue by smxsm** [teracyhq/dev#304]
Hi, first of all, thanks for your dev solution, works really great so far! :)
One problem I've encountered is that I can't specify **"rsync__args"** in my _"vagrant_config_override.json"_ by default, because in the Vagrantfile, there is a **"strip()"** call on the value, and since "rsync_options" is supposed to be an array, that fails :(

`"rsync__args": ["--verbose", "--archive", "-z", "--copy-links"]`

Leads to:

> Vagrantfile:182:in `block (2 levels) in <top (required)>': undefined method `strip' for ["--verbose", "--archive", "-z", "--copy-links"]:Array (NoMethodError)

If I remove the strip(), all is fine:

```
      #options[:rsync__args] = item['rsync__args'] unless item['rsync__args'].nil? or item['rsync__args'].strip().empty?

      options[:rsync__args] = item['rsync__args'] unless item['rsync__args'].nil? or item['rsync__args'].empty?
```
Am I doing something wrong or is that a bug? :)

**Comment by hoatle** [comment]
@smxsm nice catch, it's a bug, could you help create a patch for it?

**Comment by smxsm** [comment]
That was fast :) sure, I'll try to add a PR here

**Comment by smxsm** [comment]
Here you go ... https://github.com/teracyhq/dev/pull/307

**Comment by hoatle** [comment]
@smxsm thank you, the fix will be available by v0.5.0-c1.
